docs/src/examples/components/Popup/Types/PopupExamplePointing.shorthand.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react'
+import { Button, Grid, Popup } from '@stardust-ui/react'
+
+const PopupWithButton = props => (
+  <Popup
+    align={props.align}
+    content="A popup with a pointer."
```

Comment:
For the other popup example, we intentionally use multiline content to clearly show the difference between `top`/`center`/`bottom` - it would be great to have the same here as well.